### PR TITLE
Ordered list

### PR DESCRIPTION
**Pull request by ksanjeev284**
Hi, I have added the Ordered List function; please review it.

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ site_description: LearnHouse is an open-source platform tailored for learning ex
 contact_email: hi@learnhouse.app
 
 general:
-  development_mode: false
```
Comment (swve) [inline config/config.yaml:6]: You pushed config changes, I believe this is an error.

**Review by swve — CHANGES_REQUESTED** [review]

**Review by swve — COMMENTED** [review]
You also need to add the extension in the DynamicCanva here : https://github.com/learnhouse/learnhouse/tree/c1f9cda15cceef48ef1e7601dc35cb0a1a8f4d5d/front/components/Pages/Activities/DynamicCanva

**Comment by swve** [comment]
Also, can you rebase with the current `dev` branch, thanks!

**Review by ksanjeev284 — COMMENTED** [review]
Hi,

I have fixed all issues, please check

**Comment by swve** [comment]
Thanks @ksanjeev284 for your contribution! 
It is merged 🎉

**Pull request merged**